TODO

```diff
@@ -47,6 +47,10 @@ Features:
 
 * RestrictNamespaces= or so in services (taking away the ability to create namespaces, with setns, unshare, clone)
 
+* RestrictRealtime= which takes aware ability to create realtime processes
+
+* nspawn: make /proc/sys/net writable?
```

Comment:
I think you could just push those kind of things directly too master, unless they're somehow related to the PR.